pkgs/top-level/all-packages.nix

```
@@ -23239,6 +23239,7 @@ with pkgs;
     buildUBoot
     ubootTools
     ubootA20OlinuxinoLime
+    ubootA20OlinuxinoLime2
```

```suggestion
    ubootA20OlinuxinoLime2EMMC
```

(See other comment))